backend/corpora/dataset_processing/download.py

```diff
@@ -4,16 +4,18 @@
 
 from backend.corpora.common.corpora_orm import DbDatasetProcessingStatus, UploadStatus
 from backend.corpora.common.entities import Dataset
-from backend.corpora.common.utils.db_utils import db_session_manager
+from backend.corpora.common.utils.db_utils import db_session_manager, processing_status_updater
 from backend.corpora.common.utils.math_utils import MB
 
 logger = logging.getLogger(__name__)
 
 
 class ProgressTracker:
-    def __init__(self, file_size: int):
+    def __init__(self, file_size: int, artifact_bucket: str, cellxgene_bucket: str):
         self.file_size: int = file_size
         self._progress: int = 0
+        self.artifact_bucket = artifact_bucket
```

Comment:
you also don't need the buckets anymore since you aren't removing anything from s3.